### PR TITLE
A bug in crossbeam v0.4.0

### PR DESCRIPTION
**Pull request by stjepang**

### CODE REVIEW

**Comment by Shnatsel** [comment]
According to the version specification here 0.3.2 is vulnerable, and that's the most popular version right now according to crates.io download statistics: https://crates.io/crates/crossbeam

Is 0.3.2 really affected?

File: crates/crossbeam/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[advisory]
+# Identifier for the advisory (mandatory). Will be assigned a "RUSTSEC-YYYY-NNNN"
```
Comment (tarcieri) [inline crates/crossbeam/RUSTSEC-0000-0000.toml]: Can you please remove the comments? Thanks!

Comment (stjepang) [reply crates/crossbeam/RUSTSEC-0000-0000.toml]: Done.

File: crates/crossbeam/RUSTSEC-0000-0000.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The flaw was corrected by wrapping elements inside queues in a
+`ManuallyDrop`.
+
+Thanks to @c0gent for reporting the issue.
+"""
+patched_versions = [">= 0.4.1"]
+unaffected_versions = ["< 0.4.0"]
+url = "https://github.com/crossbeam-rs/crossbeam-epoch/issues/82"
+keywords = ["concurrency", "memory-management"]
```
Comment (tarcieri) [inline crates/crossbeam/RUSTSEC-0000-0000.toml]: Though these keywords are freeform, there are a few other advisories tagged `memory-corruption` so I might advise that one here.

**Comment by tarcieri** [comment]
Looks good, thanks!

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2018-0009` in #76

**Pull request merged**